test/unit/connectors/CustomsDeclareExportsMovementsConnectorSpec.scala

```diff
@@ -305,16 +304,16 @@ class CustomsDeclareExportsMovementsConnectorSpec extends ConnectorSpec {
 
     "received InternalServerError (500) response" should {
 
-      "return failed Future" in {
+      "return Internal server error" in {
 
         stubFor(
           get(s"/consignment-query/$conversationId?eori=eori")
             .willReturn(aResponse().withStatus(INTERNAL_SERVER_ERROR))
         )
 
-        intercept[Upstream5xxResponse] {
-          await(connector.fetchQueryNotifications(conversationId, "eori"))
-        }
+        val response = connector.fetchQueryNotifications(conversationId, "eori").futureValue
+
+        response.status mustBe INTERNAL_SERVER_ERROR
```

Comment:
There is no exception to be thrown, `httpVerbs` changed the implementation (Reference: https://github.com/hmrc/http-verbs#migration)

The new version of `httpVerbs` is coming to us with `bootstrap-play-26`